src/main/scala/loamstream/drm/lsf/BacctAccountingClient.scala

```diff
@@ -0,0 +1,185 @@
+package loamstream.drm.lsf
+
+import java.time.Instant
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeFormatterBuilder
+import java.time.temporal.ChronoField
+
+import scala.util.Success
+import scala.util.Try
+import scala.util.matching.Regex
+
+import loamstream.conf.LsfConfig
+import loamstream.drm.AccountingClient
+import loamstream.drm.Queue
+import loamstream.model.execute.Resources.LsfResources
+import loamstream.model.quantities.CpuTime
+import loamstream.model.quantities.Memory
+import loamstream.util.Loggable
+import loamstream.util.Options
+import loamstream.util.RetryingCommandInvoker
+import loamstream.util.Tries
+
+/**
+ * @author clint
+ * Apr 18, 2019
+ */
+final class BacctAccountingClient(
```

Comment:
Just curious what the "B" in `Bacct...` stands for and if it's worth spelling out or putting in a comment so that it's easy to find in LSF/EBI documentation if needed?

Reply:
I have no idea.  All the Uger commands start with `q` (maybe short for "queue"?) and all the LSF ones start with `b`.